config/promitor/resource-discovery/resource-discovery-declaration.yaml

```diff
@@ -53,6 +53,8 @@ resourceDiscoveryGroups:
   type: NetworkInterface
 - name: postgres-databases
   type: PostgreSql
+- name: powerbi-dedicated
+  type: PowerBiEmbedded
```

Comment:
As discussed, we should use `PowerBiDedicated` everywhere

Reply:
fixed